### ISSUE TITLE
rn50v115_tf 

### ISSUE THREAD

**Issue by jeff-yajun-liu** [NVIDIA/DeepLearningExamples#834]
build docker container: 
https://github.com/NVIDIA/DeepLearningExamples/tree/master/TensorFlow/Classification/ConvNets/resnet50v1.5
run synthetic benchmark: 
"python ./main.py --mode=training_benchmark --warmup_steps 200 --batch_size 64 --num_iter 100 --use_xla --use_dali --results_dir=/data/tfrecords/log_example" for synthetic benchmarking the results: "DLL 2021-02-18 06:21:29.862595 - () train_throughput : 0". why throughput is "0". 
It only works when data_dir points to the imageNet dataset.

**Comment by hXl3s** [comment]
Hi @jeff-yajun-liu 

This issue is mostly because number of iteration required in benchmark is smaller than the number of warmup steps
For better stability, performance is measured after warmup steps. On the other hand, to do not repeat computation in training iteration count includes warmup steps. When the number of iterations is lower than warmup there is no data to measure throughput and by design it return 0.

When data_dir points to imageNet then the num_iter means number of epoch (each with several batch iterations inside), so go over 200 warmup steps.

Could you please test it with warmup_steps < num_iter and check if issue is still there? For instance `python ./main.py --mode=training_benchmark --warmup_steps 50 --batch_size 64 --num_iter 100 --use_xla --use_dali --results_dir=/data/tfrecords/log_example`?

Also in case on tensorflow, dali backend is usually 10-15% slower than the native TF one for V100 GPU. You should get much better results without --use_dali flag.

**Comment by jeff-yajun-liu** [comment]
That worked, thanks!
